Lib/asyncio/queues.py

```diff
@@ -112,7 +134,11 @@ async def put(self, item):
 
         Put an item into the queue. If the queue is full, wait until a free
         slot is available before adding item.
+
+        Raise a QueueShutDown if _shutdown_state is not set to `ALIVE`.
```

Comment:
```suggestion
        Raises QueueShutDown if the queue has been shut down.
```

Doc-strings are for humans